cpp/src/parquet/statistics.cc

```diff
@@ -527,9 +616,28 @@ class TypedStatisticsImpl : public TypedStatistics<DType> {
 
   void IncrementNumValues(int64_t n) override { num_values_ += n; }
 
+  static bool IsMeaningfulLogicalType(LogicalType::Type::type type) {
```

Comment:
Emm... May I ask why only F16 is the meaningFul logical type?

Reply:
This just happens to be first logical type that dramatically affects how the physical value (a `FLBA` in this case) should be interpreted for comparisons. But I tried to generalize things in case any similar logical types are added in the future.

Reply:
Thanks for you explanation!